pkg/logs/status/warning_test.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package status
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type mockConcreteWarning struct {
+	id int
+}
+
+func (c mockConcreteWarning) Render() string {
+	return "foo"
+}
+
+func TestRaise(t *testing.T) {
```

Comment:
I'd rather not test the inner map of the `warnings` struct but just that `Raise` followed by a `Get` returns what I expect